fs/dafsclient/rmtclient.cpp

```diff
@@ -79,8 +79,26 @@ void cleanupDaFsSocket(ISocket *sock)
 ///////////////////////////
 
 
-static unsigned maxConnectTime = 0;
-static unsigned maxReceiveTime = 0;
+static const unsigned defaultDafsConnectTimeoutSeconds=100;
+static const unsigned defaultDafsConnectRetries=2;
+static const unsigned defaultDafsMaxRecieveTimeSeconds=0;
+static const unsigned defaultDafsConnectFailRetrySeconds=10;
```

Comment:
As per JIRA/PR comment, this reduces the default down to 10 seconds, previously it was 15 minutes (DAFS_CONNECT_FAIL_RETRY_TIME)